test/parallel/test-event-emitter-errors.js

```diff
@@ -1,14 +1,23 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const EventEmitter = require('events');
-const assert = require('assert');
 
 const EE = new EventEmitter();
 
-assert.throws(() => {
-  EE.emit('error', 'Accepts a string');
-}, /^Error: Unhandled "error" event\. \(Accepts a string\)$/);
+common.expectsError(
+  () => EE.emit('error', 'Accepts a string'),
+  {
+    code: 'ERR_UNHANDLED_ERROR',
+    type: Error,
+    message: 'Unhandled error. (Accepts a string)'
```

Comment:
The message looks somewhat weird here because of the dot before the brackets.
I think it would be fine to either remove the dot or to move it to the end of the string.